plugindocs.rb

```diff
@@ -71,7 +71,7 @@ def execute
         # write the doc
         File.write(output_asciidoc, content)
         puts "#{plugin.canonical_name}@#{plugin.tag}: #{release_date}\n"
-      end
+      end unless released_plugin.only_publish_default? && !is_default_plugin
```

Comment:
I would prefer to do this skipping early instead of hiding it in a tailing `unless` clause.

~~~ diff
diff --git a/plugindocs.rb b/plugindocs.rb
index 819b287..7074fb7 100644
--- a/plugindocs.rb
+++ b/plugindocs.rb
@@ -44,6 +44,11 @@ class PluginDocs < Clamp::Command
                          "unreleased"
       changelog_url = released_plugin.changelog_url
 
+      if released_plugin.type == 'integration' && !is_default_plugin
+        $stderr.puts("[repository:#{repository_name}]: Skipping non-default Integration Plugin\n")
+        next
+      end
+
       released_plugin.with_embedded_plugins.each do |plugin|
         $stderr.puts("#{plugin.desc}: fetching documentation\n")
         content = plugin.documentation
~~~
~~~